src/components/transactions/Supply/SupplyModalContent.tsx

```diff
@@ -220,15 +220,30 @@ export const SupplyModalContent = ({
         poolReserve.symbol === 'AAVE' &&
         isFeatureEnabled.staking(currentMarketData) && <AAVEWarning />}
       {poolReserve.symbol === 'SNX' && !maxAmountToSupply.eq('0') && <SNXWarning />}
-      {(currentNetworkConfig.name === 'Harmony' || currentNetworkConfig.name === 'Fantom') && (
+      {currentNetworkConfig.name === 'Harmony' ? (
```

Comment:
All of the warnings in this file can be removed. This warning was in place when borrowing was disabled by supply was enabled in the Harmony market. If both markets are disabled, the supply modal is unreachable for all assets